frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -108,6 +108,10 @@ export default function AddUser(): JSX.Element {
       },
       isCreatingUser: true
     });
+    Emitters.generalSuccess.emit({
```

Comment:
Thinking about this more, I believe this alert is redundant, since a new "Edit User" window pops up when we hit "Create user." I'd remove this one. I'm contradicting what I wrote in the ticket. My fault!